content/en/opentelemetry/guide/otel_api_tracing_interoperability.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Interoperability of OpenTelemetry API and Datadog instrumented traces
+kind: guide
+further_reading:
+- link: "/tracing/trace_collection/otel_instrumentation/"
+  tag: "Documentation"
+  text: "Custom Instrumentation with OpenTelemetry API"
+- link: "/tracing/trace_collection/trace_context_propagation/"
+  tag: "Documentation"
+  text: "Trace Context Propagation in Datadog"
+
+---
+
+## Overview
```

Comment:
One last little bit of non-blocking feedback: If we end up making this page more of a landing page to indicate how Datadog tracing libraries interoperate with OpenTelemetry components, then we could rename this overview section to a section like `Custom instrumentation with the OpenTelemetry API`